distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc

```diff
@@ -116,19 +116,15 @@ Prior to installing ${branding}, ensure the system time is accurate to prevent f
 
 === Quick Install of ${branding}
 
-.Check System Time
-[WARNING]
-====
-Prior to installing ${branding}, ensure the system time is accurate to prevent federation issues.
-====
-
 . Download the ${branding} {download-url}[zip file].
 . Install ${branding} by unzipping the zip file.
 +
 .Windows Zip Utility Warning
 [WARNING]
 ====
-The default Windows zip utility (such as double-clicking file) will not work to unzip the distribution, use Java or a third party utility instead.
+The Windows Zip implementation, which is invoked when a user double-clicks on a zip file in the Windows Explorer, creates a corrupted installation.
```

Comment:
Yeah, it's a borderline. It's such a small detail that I didn't carve it out. So much of this quickstart is a simplified version of the `Managing` section that I just assume changing it in one place requires at least a look at the other. I worry that chopping this up too small might be a management burden as well.